core/src/main/scala/com/typesafe/tools/mima/core/Problems.scala

```diff
@@ -41,7 +41,7 @@ sealed abstract class Problem extends ProblemRef {
     case ReversedMissingMethodProblem(ref)                => s"${ref.memberString} is present only in $affectedVersion version"
     case FinalMethodProblem(ref)                          => s"${ref.methodString} is declared final in $affectedVersion version"
     case IncompatibleResultTypeProblem(ref, newmeth)      => s"${ref.methodString} has a different result type in $affectedVersion version, where it is ${newmeth.tpe.resultType} rather than ${ref.tpe.resultType}"
-    case IncompatibleSignatureProblem(ref, newmeth)       => s"${ref.methodString} has a different signature in $affectedVersion version, where it is ${orMIA(newmeth.signature)} rather than ${orMIA(ref.signature)}"
+    case IncompatibleSignatureProblem(ref, newmeth)       => s"${ref.methodString} has a different signature in $affectedVersion version, where it is ${orMIA(newmeth.signature)} rather than ${orMIA(ref.signature)}. Note that the erased signature is still consistent. This means this change in the generics may or may not lead to problems at runtime, and possibly can be safely filtered out"
```

Comment:
Hmm, the message is already rather long, adding a link would make it huge... would it make sense to *just* link to the README? Like:

```suggestion
    case IncompatibleSignatureProblem(ref, newmeth)       => s"${ref.methodString} has a different generic signature in $affectedVersion version, where it is ${orMIA(newmeth.signature)} rather than ${orMIA(ref.signature)}. See https://github.com/lightbend/mima#incompatiblesignatureproblem"
```
?